roadmap/implementors-guide/src/types/backing.md

```diff
@@ -0,0 +1,81 @@
+# Backing Types
+
+[Candidates](candidate.html) go through many phases before being considered included in a fork of the relay chain and eventually accepted.
+
+These types describe the data used in the backing phase. Some are sent over the wire within subsystems, and some are simply included in the relay-chain block.
+
+## Validity Attestation
+
+An attestation of validity for a candidate, used as part of a backing. Both the `Seconded` and `Valid` statements are considered attestations of validity. This structure is only useful where the candidate referenced is apparent.
+
+```rust
+enum ValidityAttestation {
```

Comment:
I don't understand when we'd prefer to use a `ValidityAttestation` over a `SignedStatement`. Just a size optimization?

Reply:
We don't want `SignedStatement::Invalid` to creep into the backing. This prevents that on the type-system level.